### DISCUSSION TITLE
Search only numeric values on mobile

### DISCUSSION THREAD

**Discussion by krajcikondra** [JedWatson/react-select#5609] [Ideas]
Hello,

I have selec with only numeric values, for example weight or height. When I am on mobile phone and open select, alphabetically keyboard will appear. I must change keyboard to numeric and type what I want to search.

It is possible add feature for appear numeric keyboard immediatelly? Some solution how to do that in html is here.

[https://stackoverflow.com/questions/6178556/phone-numeric-keyboard-for-text-input](https://stackoverflow.com/questions/6178556/phone-numeric-keyboard-for-text-input)

Thanks in adanvce

**Comment by manjushsh** [comment]
Hi @krajcikondra,
Here is a solution for requirement with custom components. [Code Sandbox Link](https://codesandbox.io/s/laughing-pare-wx2frg?file=/src/App.js)

**Reply by krajcikondra** [reply]
Thanks a lot
